test/unit_tests/util/PathTest.cpp

```diff
@@ -10,6 +10,8 @@
  */
 
 
+#include <filesystem>
+
```

Comment:
That should be reverted, or we should choose, to abandon compilers / STL implementations which do not provide `std::filesystem`